docker/pipelines_runner/pipelines_runner.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright 2019 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+readonly google_cloud_project="${GOOGLE_CLOUD_PROJECT:-$(gcloud config get-value project)}"
+readonly temp_location="${TEMP_LOCATION}"
```

Comment:
Do users typically want to inspect the temporary files?  Ideally we would just build a bucket, set a reasonable TTL and delete it when we are done.

This isn't something we should do in this change but would be good to think about.

Reply:
Thank you, this is a good point and removing it can simplify the running of our tool. I have created an issue ([issue 442](https://github.com/googlegenomics/gcp-variant-transforms/issues/442)) for it.

Reply:
BTW, the `temp_location` provided here is for pipelines logging. And the user doesn't need to provide one.